index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      body {
+        background-color: pink;
+      }
+      
+      @media screen and (min-width: 480px) {
+        body {
+          background-color: lightgreen;
+        }
+      }
+      </style>
+  </head>
+  
+  <body>
+
+    <nav id="navbar">
+      <a href="#welcome-section" id="profile-link" target="_blank">projects</a>
+     </nav>
+     <section id="welcome-section" name="projects">
+      <h1>Test</h1>
+      <h1>hello</h1>
```

Comment:
It would be better if you use one of the `<h1>` tags on your page this should represent the main heading/subject for the whole page. The heading levels - start with `<h1>` then better to use `<h2>`